sample_apps/run_example.sh

```diff
@@ -1045,22 +1069,32 @@ function construct_policyKey_platform_is_trusted_simple_app_under_app_service()
     construct_policyKey_platform_is_trusted_app
 }
 
+# ###########################################################################
+function construct_policyKey_platform_is_trusted_simple_app_under_keystone() {
+    construct_policyKey_platform_is_trusted_app
+}
+
 # ###########################################################################
 # Work-horse function shared between different apps.
 # ###########################################################################
 function construct_policyKey_platform_is_trusted_app() {
     run_cmd
     run_pushd "${PROV_DIR}"
 
+    local subject_arg="--key_subject"
     local key_subject_file="platform_key_file.bin"
 
     # The app and App Service should share the same policy_key_file.bin.
     if [ "${SampleAppName}" = "simple_app_under_app_service" ]; then
         key_subject_file="policy_key_file.bin"
+
+    elif [ "${SampleAppName}" = "simple_app_under_keystone" ]; then
+        subject_arg="--cert_subject"
+        key_subject_file="emulated_keystone_key_cert.bin"
```

Comment:
Somehow, as per the steps in `temporary_instructions.txt`, we need to issue the clause as:

```--cert_subject=emulated_keystone_key_cert.bin```

for this app. This change produces the required clause.